src/main.js

```diff
@@ -119,12 +66,64 @@ function generateUserCover() {
   descriptors.push(userDescriptor1, userDescriptor2)
 }
 
+function goHome() {
+  homeView.classList.remove('hidden');
```

Comment:
It might be good to add some sort of helper function to clean up this function! Maybe use a toggle here?